src/utilities.ts

```diff
@@ -9,15 +9,13 @@ export type Is<T1, T2> =
     (<T>() => T extends T2 ? true : false) extends (<T>() => T extends T1 ? true : false)
         ? true
         : false;
-
-// returns true if T1 is a subtype of T2 (read: T1 is assignable to T2)
 export type Extends<A1, A2> =
-    [A1] extends [never] ? true :
-        [A2] extends [never] ? false :
-            [any] extends [A2] ? true :
-                [unknown] extends [A2] ? true :
-                    [any] extends [A1] ? false :
-                        [unknown] extends [A1] ? false :
```

Comment:
Beside there was a bug ☝️ the [bracket] notion prevents unions from spreading.
The new version 👇 is using the `Is` utility instead.